clarity/data/HOA_tools_cec2.py

```diff
@@ -145,74 +145,80 @@ def P(i, a, b, el, r):
 
 
 @njit
-def U(m, n, el, r):
+def U(degree, n, order, rotation_matrices):
     """U coefficient initialiser for rotation matrix calculation.
 
     Args:
-        m (int): degree
+        rotation_degree (int): Number of degrees to rotate by.
         n (int): index
-        el (int): order
-        r (list(matrix)): rotation matrices
+        order (int): order
+        rotation_matrices (list(matrix)): rotation matrices
 
     Returns:
         float: U value
     """
-    return P(0, m, n, el, r)
+    return P(0, degree, n, order, rotation_matrices)
 
 
 @njit
-def V(m, n, el, r):
+def V(degree, n, order, rotation_matrices):
     """V coefficient initialiser for rotation matrix calculation.
 
     Args:
-        m (int): degree
+        degree (int): degree valid values are 0, 1 and -1.
```

Comment:
the valid values for this would only be -1, 0, 1 for the case of the first order rotation submatrix.
for higher orders, which this code computes, valid inputs are int(|m|)<=el